packages/dds/cell/src/cell.ts

```diff
@@ -278,7 +285,13 @@ export class SharedCell<T = any> extends SharedObject<ISharedCellEvents<T>>
         return this.serializer.decode(value);
     }
 
-    protected applyStashedOp() {
-        throw new Error("not implemented");
+    /**
+     * {@inheritDoc @fluidframework/shared-object-base#SharedObjectCore.applyStashedOp}
+     * @internal
+     */
+    protected applyStashedOp(content: any) {
```

Comment:
I think it actually might be better to use `unknown` instead of `any` here, since there is an implicit `any` -> `ICellOperation` assertion when we access message.contents in applyInnerOp(). With unknown you would have to explicitly assert this which would be more clear